pyquil/quil.py

```diff
@@ -980,20 +1015,19 @@ def merge_with_pauli_noise(prog_list: Iterable, probabilities: List, qubits: Lis
     return p
 
 
-def merge_programs(prog_list):
+def merge_programs(prog_list: List[Program]) -> Program:
```

Comment:
```suggestion
def merge_programs(prog_list: Sequence[Program]) -> Program:
```

Just an `Iterable` would do, but `Sequence` seems like a reasonable backwards/forwards compatible compromise.

Reply:
Fixed in 5th commit.